moma_controllers/moma_ocs2_ros/include/moma_ocs2_ros/mpc_velocity_controller.h

```diff
@@ -342,10 +349,10 @@ class MpcController {
   }
 
   // path processing
-  static bool sanityCheck(const nav_msgs::Path& path) {
+  bool sanityCheck(const nav_msgs::Path& path) {
     // check that path adheres to conventional frame
-    if (path.header.frame_id != "world") {
-      ROS_ERROR("[MpcController::transformPath] Desired path must be in world frame.");
+    if (path.header.frame_id != referenceFrame_) {
+      ROS_ERROR_STREAM("[MpcController::transformPath] Desired path must be in [" << referenceFrame_ << "] frame. This should coincide at initialization, with the base link.");
```

Comment:
Maybe more general wording:
"... with the first frame of the MPC robot model."

Reply:
Yes it wasnt so clear, but I rather changed to the "root" of the kinematic chain because that is what the first frame is.